doc/src/modules/polys/domainmatrix.rst

```diff
@@ -0,0 +1,43 @@
+.. _polys-domainmatrix:
+
+===============================================
+Introducing the domainmatrix of the poly module
```

Comment:
@ks147, this was pushed by mistake in the PR, This is not a finished work, as, I was working on it. if you feel like adding it, then I can add some more material to it.
Kindly tell.
Thanks

Reply:
Yes, Please complete the PR.